### ISSUE TITLE
About eval data curve

### ISSUE THREAD

**Issue by unparalleled-ysj** [TensorSpeech/TensorFlowTTS#438]
@dathudeptrai Hi, I have a question. Why does the training data loss curve look steady downward when training the fastspeech2 v2 model, but the eval data loss curve oscillates so much?
<img width="1064" alt="截屏2020-12-22 下午5 12 38" src="https://user-images.githubusercontent.com/39238619/102871205-1b9df380-4479-11eb-8df2-c300e1eeba34.png">
<img width="1063" alt="截屏2020-12-22 下午5 12 56" src="https://user-images.githubusercontent.com/39238619/102871226-20fb3e00-4479-11eb-82d1-1770b9849a66.png">

**Comment by dathudeptrai** [comment]
> @dathudeptrai Hi, I have a question. Why does the training data loss curve look steady downward when training the fastspeech2 v2 model, but the eval data loss curve oscillates so much?
> <img alt="截屏2020-12-22 下午5 12 38" width="1064" src="https://user-images.githubusercontent.com/39238619/102871205-1b9df380-4479-11eb-8df2-c300e1eeba34.png">
> <img alt="截屏2020-12-22 下午5 12 56" width="1063" src="https://user-images.githubusercontent.com/39238619/102871226-20fb3e00-4479-11eb-82d1-1770b9849a66.png">

that's ok. You can see that the vertical value of eval loss curves is very small that is why you see it oscillates so much :D.
